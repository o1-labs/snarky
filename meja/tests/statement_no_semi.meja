let x = 15
let y = 18
