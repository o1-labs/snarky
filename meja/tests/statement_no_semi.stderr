File "tests/statement_no_semi.meja", line 2, characters 0-3:
Error: Missing semicolon.
